docs/020-deployment/howtos/111-aks-upgrade.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: cluedin
+nav_order: 13
+parent: How-to guides for PaaS
+grand_parent: Installation
+permalink: /deployment/infra-how-tos/aks-upgrade
+title: AKS upgrade
+tags: ["deployment", "ama", "marketplace", "azure", "aks", "kubernetes", "upgrade"]
+last_modified: 2024-03-27
+---
+## On this page
+{: .no_toc .text-delta }
+1. TOC
+{:toc}
+
+In this article, you will learn how to upgrade your Azure Kubernetes Service (AKS) to a supported version.
+This is included as part of the AMA agreement with us and we'll normally facilitate these upgrades on your behalf, however, there may be times where CluedIn are unable to do this upgrade due to missing levels of permission or other factors. 
+
+{:.important}
+Before starting the AKS upgrade, make sure that you have read the pre-requisites section.
+
+## Pre-requisites
+
+### Tools used
+- `az cli` and a shell
+
+    This guide will primarily use `az cli` along with `PowerShell` to manage the AKS cluster. It is recommended to follow along, but not necessarily required.
+
+### Azure checks
+When doing a Kubernetes upgrade, nodes will `surge` to try minimise downtime of the application. This means that additional Nodes in the `Node Pool` will be spun up at upgrade time and may cause issues if preparation isn't done beforehand.
+
+- Quota
+
+    Each node will eventually spin up a new replica of the same SKU. It's important to make sure you have additional allocated quota so that these nodes can be spun up without any problems
```

Comment:
```suggestion
- **Quota** – each node will eventually spin up a new replica of the same SKU. It's important to make sure you have additional allocated quota so that these nodes can be spun up without any problems.
```